### PR TITLE
Remove logging from server

### PR DESCRIPTION
**Pull request by auchenberg**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
+// @ts-check
 var express = require('express');
 var path = require('path');
-var favicon = require('serve-favicon');
+// var favicon = require('serve-favicon');
 var logger = require('morgan');
```
Comment (rebornix) [inline index.js]: remove.

File: index.js [hunk 2]
```diff
@@ -18,9 +19,8 @@ hbs.registerPartials(__dirname + '/views/partials');
 app.set('view engine', 'hbs');
 
 // uncomment after placing your favicon in /public
-app.use(favicon(path.join(__dirname, 'public', 'favicon.ico')));
+// app.use(favicon(path.join(__dirname, 'public', 'favicon.ico')));
 
-app.use(logger('dev'));
```
Comment (rebornix) [inline index.js]: really, we want to remove the logger?

File: index.js (hunk 1 above)
Comment (rebornix) [inline index.js]: remove.

Comment (AmandaSilver) [inline index.js]: test

Comment (AmandaSilver) [inline index.js]: test

**Comment by AmandaSilver** [comment]
small nit

File: index.js [hunk 3]
```diff
@@ -18,9 +19,8 @@ hbs.registerPartials(__dirname + '/views/partials');
...
 app.use(bodyParser.json());
```
Comment (rebornix) [inline index.js]: Do we really want to use `json()`?

File: index.js (hunk 1 above)
Comment (AmandaSilver) [inline index.js]: I think we should remove this variable.

**Comment by AmandaSilver** [comment]
This is a minor nit...

File: index.js (hunk 1 above)
Comment (AmandaSilver) [inline index.js]: Can we remove this variable?

**Comment by AmandaSilver** [comment]
One small nit.

File: index.js (hunk 1 above)
Comment (AmandaSilver) [inline index.js]: Is this even needed anymore?

**Comment by AmandaSilver** [comment]
LGTM. Just one little nit, TS picked up.

File: index.js (hunk 1 above)
Comment (AmandaSilver) [inline index.js]: Do we even need this anymore?

**Comment by AmandaSilver** [comment]
LGTM. Just one nit that TypeScript picked up!

File: index.js (hunk 1 above)
Comment (AmandaSilver) [inline index.js]: Do we even need this anymore?

**Comment by AmandaSilver** [comment]
LGTM. One nit TypeScript picked up!

File: index.js (hunk 1 above)
Comment (AmandaSilver) [inline index.js]: Do we even need this anymore?

**Comment by AmandaSilver** [comment]
LGTM. One little nit TS picked up.

File: index.js (hunk 1 above)
Comment (AmandaSilver) [inline index.js]: Linter picked up unused var.

**Comment by AmandaSilver** [comment]
LGTM. One little nit. TS picked up.

File: index.js (hunk 1 above)
Comment (rebornix) [reply index.js]: yes, I should remove it.

File: index.js (hunk 3 above)
Comment (rebornix) [reply index.js]: no.

**Pull request closed without merging**